src/tests/components/LocationDetails.test.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import LocationDetails from "../components/LocationDetails";
```

Comment:
This is failing because the component cant be found, the import statement is targeting the wrong directory
```
import LocationDetails from "../../components/LocationDetails";
```